@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/mass/fn_massSet.sqf

```diff
@@ -20,14 +20,14 @@ params ["_heli"];
 
 private _emptyMass = 0;
 if (typeOf _heli == "fza_ah64d_b2e") then {
```

Comment:
We have moved away from this way of checking if the FCR is on. Instead check if the `fcr_enable` animation phase is 1.

Also this could now change mid-game through someone in zeus modifying it. Maybe you could add something to the [`AnimChanged`](https://community.bistudio.com/wiki/Arma_3:_Event_Handlers#AnimChanged) event handler to change the empty weight.



```sqf
_heli animationPhase "fcr_enable" == 1
```